README.md

```diff
@@ -14,9 +14,15 @@ pip install logzio-python-handler
 ```
 
 ## Tested Python Versions
+<table><tr><th>
+
+### Deprecation announcement
```

Comment:
I feel like this table should be at the top. @imnotashrimp WDYT?

Reply:
Yes. I understand why it should be here, and maybe a smaller note that reiterates the deprecation notice makes sense here — but this is a big enough deal that it should be right under the first H1.

Reply:
Moved it, let me know if it's ok :)